functions/utils/getChickens.js

```diff
@@ -37,6 +37,7 @@ const potentialChickens = [
   { id: "UGWEUTNH5", name: "Izzy", team: "apollo" },
   { id: "U0ZFZGJVA", name: "ray.grasso" },
   { id: "U025K202K", name: "notahat", team: "mercury" }
```

Comment:
I think this is a synxtax error since L39 doesn't end with a comma.

```suggestion
  { id: "U025K202K", name: "notahat", team: "mercury" },
```